### ISSUE TITLE
Tutorials - General issue

### ISSUE THREAD

**Issue by HerinHentry** [w3c/wai-tutorials#421]
This is a common issue across the tutorials.
When I direct the developers or any other team members to the tutorials, the question I am asked is "Is there any way to find what is a Level A, Level AA technique related fix"

I think Aria-current comes into the 2.4.8 Location AAA and may not be mandatory for Level AA.

Should we consider about providing information about the WCAG Levels as well?

**Comment by yatil** [comment]
Hi @HerinHentry,

the tutorials focus on practical guidance, most solutions are cross-levels and cannot be assigned to a specific level.

I wonder what @nitedog thinks about this issue.

(_Personally_ I find 2.4.8 so easy to archive most of the time that I have a hard time to justify why one should not do it.)

**Comment by nitedog** [comment]
The tutorials are written holistically as best-practice approach, rather than the minimum requirement for specific WCAG conformance levels. Each tutorial page includes a section listing the individual success criteria (including their levels) and techniques used, and linking to the [WCAG Quick Reference Guide](https://www.w3.org/WAI/WCAG20/quickref/). Developers can use these resource to adapt their approach with alternative techniques, as needed.

Maybe we should add something brief to that extent in the [Relationship to WCAG 2.0 intro section](https://www.w3.org/WAI/tutorials/#relationship-to-wcag-20).

**Comment by HerinHentry** [comment]
Thanks @yatil and @nitedog

**Comment by yatil** [comment]
@HerinHentry @nitedog I have made some proposals on how to address this in 296bb56c4506bf1a3f85d56b24e8429860e9d4b9.

**Comment by HerinHentry** [comment]
Thank you @yatil . Updates made to 296bb56 looks good. Tutorials cannot be the sole reference and the current statement emphasize this.
